algorithms/unit_tests/TestRandom.hpp

```diff
@@ -509,5 +579,14 @@ TEST(TEST_CATEGORY, Random_XorShift1024_0) {
       .run();
 }
 
+TEST(TEST_CATEGORY, Multi_streams) {
```

Comment:
(not formatted)
please add something like
```suggestion
TEST(TEST_CATEGORY, Multi_streams) {
#ifdef KOKKOS_ENABLE_OPENMPTARGET
    if constexpr (std::is_same_v<ExecutionSpace, \
                                 Kokkos::Experimental::OpenMPTarget>) {
      GTEST_SKIP() << "Libomptarget error"; // FIXME_OPENMPTARGET
    }
#endif
```

```

Reply:
Make sure `ExecutionSpace` is defined before this block or use directly `TEST_EXECSPACE`.